index.js

```diff
@@ -272,7 +272,7 @@ MediumClient.prototype._makeRequest = function (options, callback) {
         var err = payload.errors[0]
         callback(new MediumError(err.message, err.code), null)
       } else if (statusType == 2) {
-        callback(null, payload.data)
```

Comment:
payload.data is only undefined for token requests, but is correct in the general case. To handle both outcomes, this can be:

`callback(null, payload.data || payload)`

(Explanation on #8)